spec/index.html

```diff
@@ -292,6 +292,7 @@ <h1>
     1. NOTE: If _rawSize_ is *undefined*, then _numSize_ will be *NaN*.
     1. If _numSize_ is *NaN*, throw a *TypeError* exception.
     1. Let _intSize_ be ! ToIntegerOrInfinity(_numSize_).
+    1. If _intSize_ < 0, set _intSize_ to 0.
```

Comment:
```suggestion
    1. Let _intSize_ be the result of clamping ! ToIntegerOrInfinity(_numSize_) between 0 and Infinity.
```

Reply:
When we're only applying one bound, clamping isn't appropriate. This is also the prevailing pattern; see for example [Array.prototype.flat](https://tc39.es/ecma262/multipage/indexed-collections.html#sec-array.prototype.flat) step 4.b (which is similarly silently coercing a negative value to zero).

Reply:
fair enough, altho maybe it's worth having a prose macro for clamping to one bound